config/rewrites.json

```diff
@@ -153,5 +153,147 @@
             "rewrite": false,
             "status": 302
         }
+    ],
+    "api.rackspace.com": [
+        {  "description": "Redirect root traffic to developer.rs.com/docs/",
+            "from": "^\\/$",
+            "to": "/docs/",
+            "rewrite": false,
+            "status": 301
+        },
+        {
+            "description": "Cloud Servers redirect",
+            "from": "^\\/api-ref.html$",
+            "to": "/docs/cloud-servers/v2/developer-guide/#api-reference",
+            "rewrite": false,
+            "status": 301
+        },
+        {
+            "description": "Cloud Servers extensions redirect",
+            "from": "^\\/api-ref-servers-ext.html$",
```

Comment:
Yeah, some experimentation reveals that all of these are also accessible [without the trailing `.html` suffix](http://api.rackspace.com/api-ref-servers-ext). I'd make it optional in the pattern too with:

```
"from": "^\\/api-ref-servers-ext(\\.html)?$"
```